packages/context/src/cache.ts

```diff
@@ -0,0 +1,133 @@
+import type { Manifest } from "@sap-ux/project-access";
+import type { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+
+import type { App, Project, YamlDetails } from "./types";
+
+type AbsoluteAppRoot = string;
+type AbsoluteProjectRoot = string;
+
+class Cache {
+  private manifest: Map<AbsoluteAppRoot, Manifest>;
+  private app: Map<AbsoluteAppRoot, App>;
+  private project: Map<AbsoluteAppRoot, Project>;
+  private CAPServices: Map<AbsoluteProjectRoot, Map<string, string>>;
```

Comment:
minor comment: I would avoid to change the case of variables starting with `CAP` and keep them lower case - otherwise it looks like a class!

`capServices` would be the better member name...